packages/components/MultipleSelectionDropdown.tsx

```diff
@@ -0,0 +1,180 @@
+import React, { useRef } from "react";
+import { TouchableOpacity, View, ViewStyle } from "react-native";
+
+import chevronDownSVG from "./../../assets/icons/chevron-down.svg";
+import chevronUpSVG from "./../../assets/icons/chevron-up.svg";
+import { BrandText } from "./BrandText";
+import { SVG } from "./SVG";
+import { PrimaryBox } from "./boxes/PrimaryBox";
+import { TertiaryBox } from "./boxes/TertiaryBox";
+import { Separator } from "./separators/Separator";
+import { SpacerColumn } from "./spacer";
+import { useDropdowns } from "../context/DropdownsProvider";
+import { CheckboxDappStore } from "../screens/DAppStore/components/CheckboxDappStore";
+import {
+  additionalRed,
+  neutral17,
+  neutral44,
+  neutral55,
+  neutral77,
+  secondaryColor,
+} from "../utils/style/colors";
+import { fontMedium14, fontSemibold14 } from "../utils/style/fonts";
+import { layout } from "../utils/style/layout";
+
+interface MultipleSelectionDropdownProps {
+  style?: ViewStyle;
+  onDropdownClosed?: () => void;
+  dropdownOptions: string[];
+  placeHolder?: string;
+  setItems: (item: string) => void;
+  items: string[];
+  label?: string;
+  sublabel?: React.ReactElement;
+}
+
+export const MultipleSelectionDropdown = ({
```

Comment:
This component is 150 lines long, maybe we can refacto it to have a more readable component ?

Reply:
I find the way, but if you have any suggestions, then please let me know.